php/api/endpoints/class-coauthors-blocks-controller.php

```diff
@@ -0,0 +1,359 @@
+<?php
+/**
+ * CoAuthor Blocks
+ * 
+ * @package CoAutors_Plus\API
```

Comment:
I thought I addressed this in https://github.com/Automattic/Co-Authors-Plus/pull/997/commits/896cab3e106d57d9a88501d9e29076dc72614f80#diff-1f9be4d511e2495a24ba092e6e68d6f528118cd20eef6a8baffe048d871b5325

by switching to `@package CoAuthors` everywhere because that's what is in the main plugin file. It looks like recent changes on develop are using `@package Automattic\CoAuthorsPlus`. Which is correct?

Reply:
CoAuthors is technically a different plugin that CoAuthors Plus extended from, so I'd like to ensure that the `Plus` part is maintained (it's not in the namespaces yet but changing that would be a breaking change, so 4.0).

The common approach is to keep the "Vendor\PackageName". Since the desired namespace will be `Automattic\CoAuthorsPlus`, let's go with that for the class-level `@package` annotations please.

Reply:
OK, thanks, updated in https://github.com/Automattic/Co-Authors-Plus/pull/997/commits/7de72f445dfc92cf012ba1e85b94e6ff3c7eb0f1